easybuild/toolchains/mpi/mpich.py

```diff
@@ -46,19 +47,18 @@ class Mpich(Mpi):
 
     MPI_LIBRARY_NAME = 'mpich'
 
+    # FIXME version-dependent? see http://www.mpich.org/static/docs/v3.1.4/www1/mpifort.html
+    MPI_COMPILER_MPIFC = 'mpif90'
+    #MPI_COMPILER_MPIFC = 'mpifort'
```

Comment:
@geimer: it seems like MPICH switched from `mpif90` to `mpifort` in v3.1.2? Any idea about MPICH2?